[2022-02-12 20:51:51,103] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: 1_load_initial_data.load_awards 2022-02-12T20:45:34.617768+00:00 [queued]>
[2022-02-12 20:51:51,212] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: 1_load_initial_data.load_awards 2022-02-12T20:45:34.617768+00:00 [queued]>
[2022-02-12 20:51:51,228] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 20:51:51,229] {taskinstance.py:881} INFO - Starting attempt 2 of 2
[2022-02-12 20:51:51,230] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 20:51:51,251] {taskinstance.py:901} INFO - Executing <Task(PostgresOperator): load_awards> on 2022-02-12T20:45:34.617768+00:00
[2022-02-12 20:51:51,260] {standard_task_runner.py:54} INFO - Started process 4408 to run task
[2022-02-12 20:51:51,313] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', '1_load_initial_data', 'load_awards', '2022-02-12T20:45:34.617768+00:00', '--job_id', '19', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/load_data.py', '--cfg_path', '/tmp/tmp9xojh1zm']
[2022-02-12 20:51:51,314] {standard_task_runner.py:78} INFO - Job 19: Subtask load_awards
[2022-02-12 20:51:51,405] {logging_mixin.py:112} INFO - Running <TaskInstance: 1_load_initial_data.load_awards 2022-02-12T20:45:34.617768+00:00 [running]> on host f261333f79b3
[2022-02-12 20:51:51,443] {postgres_operator.py:62} INFO - Executing: 
                      COPY dbt_raw_data.awards 
                      FROM '/sample_data/raw__awards.csv'
                      DELIMITER ',' CSV HEADER;
                      
[2022-02-12 20:51:51,463] {base_hook.py:89} INFO - Using connection to: id: dbt_postgres_instance_raw_data. Host: postgres-dbt, Port: 5432, Schema: dbtdb, Login: dbtuser, Password: XXXXXXXX, extra: None
[2022-02-12 20:51:51,477] {dbapi_hook.py:176} INFO - 
                      COPY dbt_raw_data.awards 
                      FROM '/sample_data/raw__awards.csv'
                      DELIMITER ',' CSV HEADER;
                      
[2022-02-12 20:51:51,485] {taskinstance.py:1150} ERROR - invalid input syntax for type date: "nan"
CONTEXT:  COPY awards, line 1577, column date: "nan"
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 984, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python3.7/site-packages/airflow/operators/postgres_operator.py", line 65, in execute
    self.hook.run(self.sql, self.autocommit, parameters=self.parameters)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 177, in run
    cur.execute(s)
psycopg2.errors.InvalidDatetimeFormat: invalid input syntax for type date: "nan"
CONTEXT:  COPY awards, line 1577, column date: "nan"

[2022-02-12 20:51:51,488] {taskinstance.py:1194} INFO - Marking task as FAILED. dag_id=1_load_initial_data, task_id=load_awards, execution_date=20220212T204534, start_date=20220212T205151, end_date=20220212T205151
[2022-02-12 20:51:55,968] {local_task_job.py:102} INFO - Task exited with return code 1
